### PR TITLE
Add ngOnDestroy & bump CSS Element Queries

### PR DESCRIPTION
**Pull request by KrickRay**
- bump [CSS Element Queries](https://github.com/marcj/css-element-queries/releases/tag/1.2.0)
- `resizeSensor.detach()`
- remove `onResized()` inside init (v. 1.2 "Trigger initially the resize sensor callback")

### CODE REVIEW

**Comment by vdolek** [comment]
Thanks ;)
